app/UpdateStores.tsx

```diff
@@ -0,0 +1,99 @@
+import { ReactElement, useState } from "react";
+import { Wrapper } from "./ui/Wrapper";
+import { greenActiveButton, greyInactiveButton } from "./ui/colourLibrary";
+import { LargeButton } from "./ui/LargeButton";
+import { StyledForm } from "./ui/StyledForm";
+import { PageHeader } from "./ui/PageHeader";
+import { FormLabel } from "./ui/FormLabel";
+import { FormGroup } from "./ui/FormGroup";
+import { FormControl } from "./ui/FormControl";
+
+export function UpdateStores(props: {
```

Comment:
can you update multiple stores?

Reply:
https://github.com/auberginearms/groceryDatabaseApp/pull/19/commits/0a34086b405b956204cf8fb9127a95fee55502fe